docs/quickstarts/packet.md

```diff
@@ -27,17 +27,19 @@ The guide uses [Amazon Route 53](https://aws.amazon.com/route53/) as a DNS provi
 
 ## Requirements
 
-* Basic understanding of Kubernetes concepts.
-* Packet account, Project ID and auth token (sometimes also referred to as [User Level API
-  key](https://www.packet.com/developers/docs/API/getting-started/)).
-* AWS account and IAM credentials (optional for Route53 DNS configuration).
-* AWS Route53 DNS Zone (registered Domain Name or delegated subdomain).
-* Terraform v0.12.x and [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct) v0.5.0
-  installed locally.
-* Local BGP enabled. More information on how to enable Local BGP for the Packet Project is found in
-  the [Packet support document](https://support.packet.com/kb/articles/bgp).
-* An SSH key pair for management access.
-* `kubectl` installed locally to access the Kubernetes cluster.
+* A Packet account with
+  [local BGP](https://www.packet.com/developers/docs/network/advanced/local-and-global-bgp/)
+  enabled.
+* A Packet project ID.
+* A Packet API key with access to the relevant project.
```

Comment:
Oh. I wasn't aware there was a technical difference there. Will fix :+1:

Reply:
Well, for sure there **was**. You can see the docs we had, in case something else is useful, [here](https://github.com/kinvolk/lokomotive-kubernetes/blob/a7b6e9bf798733f4793bdfcbbc0c541d5326aaac/docs/flatcar-linux/packet.md#packet-1):
> Obtain an API Key from the User settings menu. Note that project level API keys don't have all the necessary permissions for this exercise

I know for a fact (i.e. run into that problem) that it didn't work with project level API key. Haven't tested recently, though, if anything changed.

Reply:
It's fixed ;-)